prime-router/metadata/fhir_mapping/hl7/message/ORU_R01.yml

```
@@ -60,17 +58,29 @@ resources:
       - .OBSERVATION.NTE
       - .SPECIMEN.SPM
 
+  - resourceName: SpecimenSource
```

```suggestion
  - resourceName: Specimen
```

I think docs say this should correspond to the FHIR resource